variables.tf

```
@@ -14,7 +14,16 @@ variable "vpc_group" { type = object({
             }))
             public_igw_subnet_groups = list(string)
             nacl_subnet_groups = optional(list(string), [])
-            vpc_endpoints = optional(map(list(string)), {})
+            vpc_endpoints = map(object({
```

change to
```suggestion
            vpc_endpoints = optional(map(object({
```
and set the default to empty map {}

---

Ok will update it